build/configs/scanners.yaml

```diff
@@ -93,6 +93,7 @@ scanners:
           - 'vbframe_file'
           - 'mso_file'
           - 'application/vnd.ms-excel'
+          - 'pdf_file'
```

Comment:
`pdf_file` is positive for `ScanExiftool` down on line 120. A different set of options are selected though.

Reply:
Nice catch guess i didn't scroll far enough :)

Reply:
updated